osfoffline/tasks/operations.py

```diff
@@ -132,6 +169,30 @@ def __init__(self, remote):
     @asyncio.coroutine
     def run(self):
         print("Delete Remote File: {}".format(self.remote))
+        db_parent = session.query(models.File).filter(models.File.osf_id == self.remote.parent.id).one()
+        path = os.path.join(db_parent.path, self.remote.name)
+
+        resp = yield from self.remote.request_session.request('DELETE', self.remote.id)
```

Comment:
This line involves talking to the API server, and it would be nice to keep all of the code that talks to the server in one place.  (all API requests encapsulated in a single client)

Can we move this logic to the `client/osf.py` file?